src/App.js

```diff
@@ -1,24 +1,49 @@
 import React from 'react';
-import { Routes, Route, Link } from 'react-router-dom';
-import Books from './components/Books';
-import Categories from './components/Categories';
+import { HashRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider, connect } from 'react-redux';
+import Categories from './component/categories/Categories';
+import store from './redux/store';
+import Navbar from './component/Navbar';
+import AddBook from './component/AddBook';
+import BookList from './component/BookList';
+import Progress from './component/Progress';
+
+const routes = [
+  {
+    path: '/',
+    name: 'book',
+    element: <BookList />,
+  },
+  {
+    path: '/categories',
+    name: 'categories',
+    element: <Categories />,
+  },
+];
 
 const App = () => (
-  <div className="App">
-    <div className="top">
-      <h1 className="title">Bookstore</h1>
-      <nav className="nav">
-        <Link className="a" to="/">Books</Link>
-        <Link className="a" to="/Categories">Categories</Link>
-      </nav>
-    </div>
-    <section className="section">
-      <Routes>
-        <Route path="/" element={<Books />} />
-        <Route path="Categories" element={<Categories />} />
-      </Routes>
-    </section>
-  </div>
+  <Provider store={store}>
+    <Router basename={process.env.PUBLIC_URL}>
+      <Navbar title="Bookstore CMS" routes={routes} />
+      <div className="container">
+        <ConnectedBookList />
+        <AddBook />
+        <Progress />
+        <Routes>
+          {routes.map(({ path, element }) => (
+            <Route path={path} element={element} key={path} />
+          ))}
+        </Routes>
+      </div>
+    </Router>
+  </Provider>
```

Comment:
- [ ] Kindly ensure that the entire UI container is styled to have full width after which you should add paddings around the books as well. 

Zeplin 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/01d1e2f0-6a3d-4798-a4f3-fa3bae3a3a2f)

your design 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/9cca0568-f7fc-4dd6-a610-545550b218d8)